### ISSUE TITLE
Update ponchon_savarit's feedliq boolean to q float/bool combo arg

### ISSUE THREAD

**Issue by ethanmolnar06** [Kyupii/CheToolbox#9]
```
  feedliq : bool  # TODO check if q is a valid substitute for this var
```

https://github.com/Kyupii/CheToolbox/blob/a9425d86e3540fd364fadae1ddf39ddd14a7f3f5/chetoolbox/separations.py#L425

**Comment by ethanmolnar06** [comment]
Since the q calculations are derived from energy balances within the McCabe-Thiel school of thought, they assume equal molar heats of vaporization, which Ponchon-Savarit exists to reject. Therefore, they cannot be treated as fact, but are instead an excellent starting point to begin the trial and error estimate of the feedline when F is neither saturated liquid nor vapor (does not lie directly on either enthalpy line).

TODO write code to start from a q value and interactively solve for the slope of the feedline, finding it's x_f* and y_f* values (intersects with the enthalpy lines to calculate R_min)

**Comment by ethanmolnar06** [comment]
closed with 60ecd7866c84465ffd55ff50298ac4bcceeaaf63
